src/infrastructure/common/utils.py

```diff
@@ -1,6 +1,16 @@
+import os
+
+
 def parse_float(value: str) -> float | str:
     try:
         _value = float(value)
         return _value
     except ValueError:
         return value
+
+
+def image_size(file):
```

Comment:
wrzuć to do osobnego pliku nwm "file_utils". W jednej z moich pr zmieniam nazwę etgo pliku na scalar_utils